web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/formidling/FormidlingRestTjeneste.java

```diff
@@ -88,4 +90,19 @@ public Response utsattStartdato(@TilpassetAbacAttributt(supplierClass = Behandli
         return responseBuilder.build();
     }
 
+    @GET
+    @Path(MOTATT_DATO_SØKNADSFRIST_PART_PATH)
+    @Produces(MediaType.APPLICATION_JSON + ";charset=utf-8")
+    @Operation(description = "Hent gjeldende mottatt dato for søknad", tags = "formidling")
```

Comment:
skal bare brukes for svp?

Reply:
Usikker på hvordan dette vil slå ut for fp. Aksjonspunktet for søknadsfrist gjelder både svp og fp...må sjekke litt på det. I første omgang er denne kun tenkt å brukes for svp. Men hva tenkte du på?

Reply:
har endret slik at url tilbys kun dersom svp